### DISCUSSION TITLE
Survey: Which sync method are you using?

### DISCUSSION THREAD

**Discussion by marcelklehr** [floccusaddon/floccus#967] [General]
In the absence of telemetry, I'd like to find out how popular the different sync methods are among users.

It would be awesome if you could click once for each method that you use. If you feel like you would like to give additional feedback, criticism or explanations, feel free to drop a comment below :)

Thanks :blue_heart: 

[![](https://api.gh-polls.com/poll/01FGKC96C78WW4TTMJCAZW71FX/Nextcloud%20Bookmarks)](https://api.gh-polls.com/poll/01FGKC96C78WW4TTMJCAZW71FX/Nextcloud%20Bookmarks/vote)
[![](https://api.gh-polls.com/poll/01FGKC96C78WW4TTMJCAZW71FX/WebDAV)](https://api.gh-polls.com/poll/01FGKC96C78WW4TTMJCAZW71FX/WebDAV/vote)
[![](https://api.gh-polls.com/poll/01FGKC96C78WW4TTMJCAZW71FX/Google%20Drive)](https://api.gh-polls.com/poll/01FGKC96C78WW4TTMJCAZW71FX/Google%20Drive/vote)

**Comment by mnalis** [comment]
oops, sorry. It seems I've added erronous `+5` on Nextcloud... <details>That gh-polls does not have duplicate voting protection, and while I whitelisted it in uBlock-origin, the score `0:0:0` jumped to `6:1:0` instead of `1:1:0` (alternatively, 5 people voted for nextcloud in that 30 seconds, but I find that unlikely) </details>

Also, `Discussions` have quite a low visibility, unless one has set to watch all activity on the project (which I'd guess very very few people do), so traffic should be driven somehow to this poll (ask people to vote after installing/upgrading extension or when adding account, in issue templates, in description at addons.mozilla.org and similar, at website https://floccus.org, etc). Also parsing existing issues (especially if they followed the issue template) might give some statistics (although obviously skewed by only showing users with problems)

**Comment by rakekniven** [comment]
I am using the bookmark app. It allowed me to drop the sync service of mozilla. Well done.

**Comment by lorddoumer** [comment]
Have been using Nextcloud bookmarks initially, but for a large collection of bookmarks a .xbel-file via WebDav is much faster and robust (thanks for the hint @marcelklehr) so I've been using this for a long time without any problems :)

**Comment by jimmymasaru** [comment]
I use Nextcloud bookmarks for syncing and WebDAV (to the same Nextcloud instance) for backing up. The downside is the backup file is overwritten so I only have one backup. I'm thinking if the WebDAV option can generate timestamped files it would be a great backup feature. Anyway, thanks for the great work!

**Reply by mnalis** [reply]
BTW  @jimmymasaru, you can already accomplish timestamped WebDav `.xbel` backups yourself easily if you're using self-hosted nextcloud, by adding shell script like the following in your `/etc/cron.daily` on your nextcloud instance (if you want daily backups, or configure `/etc/cron.d` for different schedules).

```
#!/bin/sh
N_USER=your_nextcloud_username
W_DIR=your_webdav_dir
cd /var/lib/nextcloud/data/${N_USER}/files/${W_DIR} || exit 1
cp -a bookmarks.xbel bookmarks-`date +%Y%m%d_%H%M`.xbel
```

**Reply by jimmymasaru** [reply]
Yeah this is a good idea. Thanks!

**Comment by al-lac** [comment]
We are still using Nextcloud Bookmarks, but would love to switch to Shared Google Drives once the Feature is implemented via #889 🙂

**Reply by rakekniven** [reply]
Using Google Drive is not supporting your data privacy.
What is the reason for you to use it?

**Reply by marcelklehr** [reply]
Some people are not so much interested in the private aspect of floccus but more in the cross-browser aspect

**Reply by rakekniven** [reply]
Cross browerser and nextcloud as backend will work as well.

**Reply by al-lac** [reply]
As @marcelklehr said, we just want it to be as easy to use for a lot of users and are not that interested in the private aspect.

**Reply by pinpontitit** [reply]
I don't see why using Google drive could be a threat to privacy if you use the built in encryption in the Floccus settings.
No one, even google, could read your bookmarks without your password.

Concerning the other files or the metadatas collected by google thats another story unrelated with Floccus

**Comment by DonPicciotto** [comment]
Nextcloud bookmarks and I am an highly satisfied user :)

**Comment by MattD-Cymru** [comment]
Nextcloud bookmarks. This is the very best solution for this sync method and I will continue to recommend it to others.

**Comment by better319** [comment]
THANKS FOR YOUR CONTRIBUTION
QR code login  is needed
so i can login my web browser by my phone camera
without having to remember the webdav password

**Comment by marcdw1289** [comment]
Using WebDAV. Working fine.
Tried Nextcloud Bookmarks but all I would get is empty directories/folders and only a subset of bookmarks on the top level.
Switched to WebDAV and all is good it seems.

I have a ROM setup where all Google and Facebook domains are blocked. It was a long while before I realized that Firefox sync wasn't working because it uses `googleusercontent.com`. That caused me to look elsewhere and Floccus caught my eye. Also used on the desktop with LibreWolf.

**Comment by pinpontitit** [comment]
There's a bug in Github for Android.
I don't see the description of each poll answers.
I mistakenly voted for the first which is Nextcloud i suppose.
But I use Google drive.

![Screenshot_20220501-000634_GitHub.jpg](https://user-images.githubusercontent.com/100489443/166124163-ff5a083e-3f70-4db1-b252-c82ade3f53af.jpg)

**Reply by pinpontitit** [reply]
LOL, when I look to this picture in my computer, I can totally see "Nextcloud, "Google drive" etc
But on my phone, you cannot read it. Maybe its my phone screen which doesnt make good level of greys?
However thats an off-topic Github dark mode issue.

**Comment by test-0-zero** [comment]
I'm using G drive but I can't chose an a specific folder in a floccus dialog box "bookmarks file path".

**Comment by test-0-zero** [comment]
G drive is the best method especially with different browsers (FF and CH) and on different operating systems (Linux and Windows) and with different accounts (google) using the sharing function of one or more folders between different accounts (google).

Looking forward to the new feature "Use alternate (shared) Google Drive location" #867

**Comment by SchwarzerA** [comment]
I'm using WebDAV for syncing XBEL-type data to a private server. Nextcloud or Google drive are no real alternatives.

Has anyone an opinion about the difference(s) between xbel (not reliable) and html (experimental)?

**Comment by hochl** [comment]
I am running rclone on a Raspbery PI and using WebDAV/XBEL. It seems to work so far.
